actors/builtin/sector.go

```diff
@@ -99,15 +65,7 @@ func SealProofWindowPoStPartitionSectors(p stabi.RegisteredSealProof) (uint64, e
 
 // SectorMaximumLifetime is the maximum duration a sector sealed with this proof may exist between activation and expiration
 func SealProofSectorMaximumLifetime(p stabi.RegisteredSealProof, nv network.Version) (stabi.ChainEpoch, error) {
```

Comment:
suggestion: remove the `nv` argument.  It is no longer needed and makes this change consistent with `CanPreCommitSealProof`